microbench/src/hpy_simple.c

```diff
@@ -65,7 +65,8 @@ static HPy allocate_tuple_impl(HPyContext *ctx, HPy self)
 /* Foo type */
 
 typedef struct {
-    PyObject_HEAD
+    long x;
+    long y;
 } FooObject;
```

Comment:
Why do we even need `FooObject`? An HPy type can also have `.basicsize = 0`.

Reply:
it's because we wanted to have non-0 allocations in `test_allocate_*`, they are more likely to be representative.